pkg/sql/pg_metadata_test.go

```diff
@@ -73,9 +73,10 @@ const (
 
 // When running test with -rewrite-diffs test will pass and re-create pg_catalog_test-diffs.json.
 var (
-	rewriteFlag = flag.Bool("rewrite-diffs", false, "This will re-create the expected diffs file")
-	catalogName = flag.String("catalog", "pg_catalog", "Catalog or namespace, default: pg_catalog")
-	rdbmsName   = flag.String("rdbms", Postgres, "Used to determine which RDBMS to compare, default: postgres")
+	rewriteFlag      = flag.Bool("rewrite-diffs", false, "This will re-create the expected diffs file")
+	catalogName      = flag.String("catalog", "pg_catalog", "Catalog or namespace, default: pg_catalog")
+	rdbmsName        = flag.String("rdbms", Postgres, "Used to determine which RDBMS to compare, default: postgres")
+	testDataFileName = flag.String("test-data-filename", "", "Location to json file to create tables manually")
```

Comment:
nit: Location to json file to create tables manually / Location to json file used to create tables manually

Reply:
Done.